central/convert/internaltov2storage/compliance_rule.go

```diff
@@ -0,0 +1,70 @@
+package internaltov2storage
+
+import (
+	"strings"
+
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/generated/storage"
+)
+
+const (
+	standardsKey = "policies.open-cluster-management.io/standards"
+
+	controlAnnotationBase = "control.compliance.openshift.io/"
+)
+
+// ComplianceOperatorRule converts message from sensor to V2 storage
+func ComplianceOperatorRule(sensorData *central.ComplianceOperatorRuleV2, clusterID string) *storage.ComplianceOperatorRuleV2 {
+	fixes := make([]*storage.ComplianceOperatorRuleV2_Fix, 0, len(sensorData.Fixes))
+	for _, fix := range sensorData.Fixes {
+		fixes = append(fixes, &storage.ComplianceOperatorRuleV2_Fix{
+			Platform:   fix.GetPlatform(),
+			Disruption: fix.GetDisruption(),
+		})
+	}
+
+	// The standards and controls that a rule applies to are stored within the annotations of the Rule CR.
```

Comment:
@theencee Is this what you were looking for?

Reply:
perfect